src/bootstrap/build/native.rs

```diff
@@ -63,7 +63,7 @@ pub fn llvm(build: &Build, target: &str) {
        .out_dir(&dst)
        .profile(if build.config.llvm_optimize {"Release"} else {"Debug"})
        .define("LLVM_ENABLE_ASSERTIONS", assertions)
-       .define("LLVM_TARGETS_TO_BUILD", "X86;ARM;AArch64;Mips;PowerPC")
+       .define("LLVM_TARGETS_TO_BUILD", "X86;ARM;AArch64;Mips;PowerPC;NVPTX")
```

Comment:
Out of curiosity, and just to confirm, this doesn't increase our binary sizes by like 100M or compile times by like 10 minutes, right? That is, this should in theory be a relatively small backend?